api/src/certification/session-management/infrastructure/repositories/jury-certification-summary-repository.js

```diff
@@ -14,8 +13,7 @@ const findBySessionId = async function ({ sessionId }) {
 
   const juryCertificationSummaryDTOs = await _getJuryCertificationSummaries(orderResults);
 
-  const juryCertificationSummaries = _.map(juryCertificationSummaryDTOs, _toDomain);
-  return juryCertificationSummaries;
+  return _.map(juryCertificationSummaryDTOs, _toDomain);
```

Comment:
```suggestion
  return juryCertificationSummaryDTOs.map(_toDomain);
```

La fonction map de lodash apporte quelque chose de plus que le natif JS ?

Reply:
aucune idée, je n'ai pas ajouté ce map